api/v1/MEL.php

```diff
@@ -260,6 +264,26 @@ public function UpdateLayer(string $layer_name): void
     // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
     public function Update(): array
     {
+        // @todo how to do this correctly ?
+        // force a mel update for all layers having a seasonal closure policy
```

Comment:
MEL calls Update() to get all the layers that need updating .
Since plans with a temporal policies might need update every month